distribution/osbuild-worker@.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=OSBuild Composer Worker (%i)
-Requires=osbuild-local-worker.socket
+Requires=osbuild-local-worker.socket osbuild-dnf-json.socket
```

Comment:
Also needs to be added in `osbuild-remote-worker@.service` and `osbuild-composer.service`; they all require dnf-json.